app/main.py

```diff
@@ -1,3 +1,72 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float = 0) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return "Distance: {} kilometers.".format(self.km)
+
+    def __repr__(self) -> str:
+        return "Distance(km={})".format(self.km)
+
+    def __add__(self, other: "Distance") -> "Distance":
```

Comment:
Indicate that other could be int, float or a `Distance` class instance. Do not use quotes for type annotation
```suggestion
    def __add__(self, other: int | float | Distance) -> Distance:
```
Apply the same changes to the other places as well. Use `from __future__ import annotations` in case you face errors

Reply:
This was not resolved